projects/ui/src/shim.cds-core.scss

```
@@ -441,23 +441,36 @@
   /* Forms */
 
   // Structure
+  // --clr-forms-baseline: var(--cds-global-space-5);
   --clr-form-disabled-background-color: var(--cds-alias-status-disabled-tint); // var(--clr-color-neutral-400);
 
   // Generic
+  --clr-forms-disabled-color: var(--cds-alias-status-disabled);
```

```suggestion
  --clr-forms-disabled-color: var(--cds-alias-typography-disabled, var(--cds-alias-status-disabled));
```

I believe this variable must be mapped to typography disabled.

---

We have both variables, one is used for text and the other for other things like color of icons borders etc..